src/test/java/marquez/api/JobRunIntegrationTest.java

```diff
@@ -0,0 +1,264 @@
+package marquez.api;
+
+import static java.lang.String.format;
+import static marquez.api.JobRunState.State.fromInt;
+import static marquez.api.JobRunState.State.toInt;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+
+import java.net.URI;
+import java.sql.Timestamp;
+import java.util.UUID;
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import marquez.db.dao.JobRunDAO;
+import marquez.db.dao.JobRunStateDAO;
+import marquez.db.dao.fixtures.DAOSetup;
+import org.junit.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class JobRunIntegrationTest {
+  private static Logger LOG = LoggerFactory.getLogger(JobRunIntegrationTest.class);
+
+  static final String TEST_JOB_GUID = UUID.randomUUID().toString();
+  static final String TEST_JOB_RUN_VERSION_GUID = UUID.randomUUID().toString();
+  static final String TEST_JOB_RUN_DEFINITION_GUID = UUID.randomUUID().toString();
+
+  @ClassRule public static final DAOSetup daoSetup = new DAOSetup();
+  final JobRunDAO jobRunDAO = daoSetup.onDemand(JobRunDAO.class);
+  final JobRunStateDAO jobRunStateDAO = daoSetup.onDemand(JobRunStateDAO.class);
+
+  private JobRun NEW_JOB_RUN;
+
+  @BeforeClass
+  public static void setUp() {
```

Comment:
TODO(code review): make the naming conform to class and test-level convention. e.g. startUp, shutDown ; setUp, tearDown